src/pdn/test/pads_black_parrot_flipchip.ok

```diff
@@ -14,4 +14,11 @@
 [INFO ODB-0133]     Created 219 nets and 267 connections.
 [INFO ODB-0134] Finished DEF file: nangate_bsg_black_parrot/floorplan_flipchip.def
 [INFO PDN-0001] Inserting grid: Core
+[WARNING PDN-0110] No via inserted between metal9 and metal10 at (174.9000, 1349.2350) - (188.5700, 1350.0000) on VSS
+[WARNING PDN-0110] No via inserted between metal9 and metal10 at (174.9000, 1829.2350) - (188.5700, 1830.0000) on VSS
+[WARNING PDN-0110] No via inserted between metal9 and metal10 at (174.9000, 2464.2350) - (195.5700, 2465.0000) on VDD
+[WARNING PDN-0110] No via inserted between metal9 and metal10 at (2804.3400, 1660.0000) - (2825.1000, 1660.7650) on VDD
+[WARNING PDN-0110] No via inserted between metal9 and metal10 at (2804.3400, 2295.0000) - (2825.1000, 2295.7650) on VDD
+[WARNING PDN-0110] No via inserted between metal9 and metal10 at (2811.3400, 535.0000) - (2825.1000, 535.7650) on VSS
+[WARNING PDN-0110] No via inserted between metal9 and metal10 at (2811.3400, 1175.0000) - (2825.1000, 1175.7650) on VSS
```

Comment:
Are these intended?

Reply:
Yes, there is no way to add the vias there without causing a DRC issue